package.json

```diff
@@ -15,11 +15,6 @@
     "type": "git",
     "url": "https://github.com/NordicSemiconductor/pc-nrfjprog-js.git"
   },
-  "author": "ken.redergard@nordicsemi.no",
-  "contributors": {
-    "name": "Per Kristian Schanke",
-    "email": "per.kristian.schanke@nordicsemi.no"
-  },
   "license": "ISC",
```

Comment:
Update license tag to "license": "SEE LICENSE IN LICENSE" (which supposedly is the correct way to point to a proprietary license)